acme.sh

```diff
@@ -4200,7 +4200,24 @@ __purge_txt() {
 _check_dns_entries() {
   _success_txt=","
   _end_time="$(_time)"
-  _end_time="$(_math "$_end_time" + 1200)" #let's check no more than 20 minutes.
+
```

Comment:
make `_dnstimeout` and `_dnsinterval` as parameters of `_check_dns_entries()` function.